WiFiManager.h

```diff
@@ -22,10 +22,10 @@ extern "C" {
   #include "user_interface.h"
 }
 
-const char HTTP_HEAD[] PROGMEM            = "<!DOCTYPE html><html lang=\"en\"><head><meta charset=\"UTF-8\" name=\"viewport\" content=\"width=device-width, initial-scale=1, user-scalable=no\"/><title>{v}</title>";
+const char HTTP_HEADER[] PROGMEM            = "<!DOCTYPE html><html lang=\"en\"><head><meta charset=\"UTF-8\" name=\"viewport\" content=\"width=device-width, initial-scale=1, user-scalable=no\"/><title>{v}</title>";
```

Comment:
That `HTTP_` is wrong/misleading/imprecise anyhow IMO. It should be `HTML_`.

Reply:
Yes, but then they all have to change also.. Ill fix it in development branch its a major ver bump